PSix/FBUserRequest.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  FBUserRequest.swift
+//  PSix
+//
+//  Created by Turzion, Avihu on 4/20/15.
+//  Copyright (c) 2015 PayPal. All rights reserved.
+//
+
+import Foundation
+
+
+class FBUserRequest: FBRequest {
+    
+    init(fbId: String) {
+        super.init(forResource: fbId)
+    }
+    
+    func about(attributes: [String]) -> FBRequest {
```

Comment:
Unfortunately swift does not support this added awesomeness for unpacking tuples or arrays :\